process_report/invoices/pi_specific_invoice.py

```diff
@@ -43,31 +70,109 @@ def _prepare(self):
         ]
         self.pi_list = self.export_data[invoice.PI_FIELD].unique()
 
+    def _get_pi_dataframe(self, data, pi):
+        def add_dollar_sign(data):
+            if pandas.isna(data):
+                return data
+            else:
+                return "$" + str(data)
```

Comment:
You don't release need the `else` clause here, and prefer f-strings over string concatenation:

```suggestion
            return f"${data}"
```

But see my comment later on about whether or not we even need this function.